htdocs/install/mysql/data/llx_c_productbatch_status.sql

```diff
@@ -0,0 +1,28 @@
+-- Copyright (C) 2021 Noé Cendrier <noe.cendrier@altairis.fr>
+--
+-- This program is free software; you can redistribute it and/or modify
+-- it under the terms of the GNU General Public License as published by
+-- the Free Software Foundation; either version 3 of the License, or
+-- (at your option) any later version.
+--
+-- This program is distributed in the hope that it will be useful,
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU General Public License for more details.
+--
+-- You should have received a copy of the GNU General Public License
+-- along with this program. If not, see <https://www.gnu.org/licenses/>.
+--
+--
+
+--
+-- Do not place a comment at the end of the line, this file is parsed when
+-- from the install and all '--' are removed.
+--
+-- Ne pas placer de commentaire en fin de ligne, ce fichier est parsé lors
+-- de l'install et tous les sigles '--' sont supprimés.
+--
+
+INSERT INTO llx_c_productbatch_status (code, label, active) VALUES ('OK', 'En état de marche', 1);
```

Comment:
Can you rename into llx_c_productbatch_qcstatus (status is reserved for had coded fields into object).

Reply:
ok

Reply:
This status will give information on the current state of an equipment, whether it is functional or broken, or, for example, in repair. This way, we know if we can sell it / rent it now or later.